security/README.md

```diff
@@ -0,0 +1,42 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Question each guideline before following it yourself and
+start a discussion whenever you're in doubt.
+
+Passwords
+---------
+
+* Use a unique password for every account you create.
+* Don't store passwords in plain text.
```

Comment:
+1 on the first comment on @mike-burns. This is confusing.

If this is for dev, I'd revise the word on 2nd bullet to say something like "don't store it in a plaintext file on your machine" or something.

If that's the case, you could also adds that you should only send password to other user via end-to-end encrypted channel (WhatsApp, iMessage, Telegram, PGP, ...)